AppServer/google/appengine/tools/devappserver2/application_configuration.py

```diff
@@ -114,22 +114,22 @@ def api_version(self):
     return self._api_version
 
   @property
-  def server_name(self):
-    return self._server_name or 'default'
+  def module_name(self):
+    return self._module_name or 'default'
 
   @property
   def major_version(self):
-    return self._version
+    return self._version or 'default' # AS: set default if not set.
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#inline-comments) recommends to put at least 2 spaces before inline-comment.